dffml/util/data.py

```diff
@@ -109,13 +131,47 @@ def traverse_get(target, *args):
     >>>
     >>> traverse_get({"one": {"two": 3}}, "one", "two")
     3
+    >>> traverse_get({"one": {"two": 3}}, "one.two")
+    3
+    >>> traverse_get({"one.two": {"three": 4}}, "'one.two'.three")
+    4
     """
+    if len(args) == 1:
+        args = split_dot_seperated(args[0])
     current = target
     for level in args:
         current = current[level]
     return current
 
 
+def traverse_set(target, *args, value):
+    """
+    Examples
+    --------
+
+    >>> from dffml import traverse_set
+    >>>
+    >>> d = {"one": {"two": 3}}
+    >>> traverse_set(d,"one.two", value = "Three")
+    >>> d["one"]["two"]
+    'Three'
```

Comment:
Test for creating levels

```suggestion
    'Three'
    >>> traverse_set(d, "newdict.'other.value'", value = 42)
    >>> d["newdict"]
    {'other.value': 42}
```